### ISSUE TITLE
TypeError: percentile() got an unexpected keyword argument 'interpolation'

### ISSUE THREAD

**Issue by bstamps** [biocore/qiime#1909]
Running filter_alignment.py produces the error

  File "/usr/local/bin/filter_alignment.py", line 4, in <module>
    __import__('pkg_resources').run_script('qiime==1.9.0-rc2', 'filter_alignment.py')
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources.py", line 517, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources.py", line 1436, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/qiime-1.9.0_rc2-py2.7.egg/EGG-INFO/scripts/filter_alignment.py", line 155, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/qiime-1.9.0_rc2-py2.7.egg/EGG-INFO/scripts/filter_alignment.py", line 148, in main
    entropy_threshold=opts.entropy_threshold):
  File "/usr/local/lib/python2.7/dist-packages/qiime-1.9.0_rc2-py2.7.egg/qiime/filter_alignment.py", line 78, in apply_lane_mask_and_gap_filter
    entropy_mask = generate_lane_mask(aln, entropy_threshold)
  File "/usr/local/lib/python2.7/dist-packages/qiime-1.9.0_rc2-py2.7.egg/qiime/filter_alignment.py", line 146, in generate_lane_mask
    interpolation='nearest')
TypeError: percentile() got an unexpected keyword argument 'interpolation'

The command was run as 

filter_alignment.py -i  otus/RepSetAligned/RepSet_aligned.fasta  -s -e 0.001 -g 0.80 -o otus/RepSetAligned/ 

This was working prior to the most recent RC2 update- unsure if this is descriptive enough, but I'm happy to provide any other information that will help!

**Comment by jairideout** [comment]
QIIME 1.9.0 will require numpy 1.9.0 or later. Can you please try upgrading numpy?

**Comment by gregcaporaso** [comment]
@bstamps, you should be able to upgrade numpy with:

```
pip install numpy --upgrade
```

**Comment by bstamps** [comment]
@jairideout @gregcaporaso Resolved- in the end here's what I had to do: 

1. Remove NumPy from my /usr/local/lib/python2.7/dist-packages/ directory
2. Reinstall NumPy using pip
3. Rebuild/Reinstall QIIME RC2

 Upgrading NumPy failed to resolve the issue prior to attempting the remove/reinstall. I attempted to only Remove/Reinstall NumPy, however print_qiime_config.py -t still showed a previous version of NumPy, and the failure within filter_alignment.py persisted.

Thank you for the help!
